src/commands/app/start-services.ts

```diff
@@ -97,35 +97,35 @@ export default class AppStartServices extends Kommand {
 
     const checks: Listr = new Listr([
       {
-        title: `docker-compose exists and the version is at least ${MIN_DOCO_VERSION}`,
+        title: `docker compose exists and the version is at least ${MIN_DOCO_VERSION}`,
         task: async () => {
           try {
-            const docov = await execute("docker-compose", "-v");
+            const docov = await execute("docker", "compose", "version");
             const matches = docov.stdout.match(/[^0-9.]*([0-9.]*).*/);
             if (matches === null) {
               throw new Error(
-                "Unable to read docker-compose verson. This is weird."
+                "Unable to read docker compose verson. This is weird."
```

Comment:
Also, maybe replace the "This is weird" with something else? Doesn't sound professional.

```suggestion
                "Unable to read the version of Docker Compose. This is weird."
```

Reply:
Well, I'll refactor all of this function and use it inside those three files, I don't like the fact that this has been duplicated everywhere